config/packages/nelmio_cors.yaml

```diff
@@ -0,0 +1,9 @@
+nelmio_cors:
+    defaults:
+        origin_regex: true
+        allow_origin: ['%env(CORS_ALLOW_ORIGIN)%']
+        allow_methods: ['GET', 'OPTIONS', 'POST', 'PUT', 'PATCH', 'DELETE']
+        allow_headers: ['Content-Type', 'Authorization']
```

Comment:
To let the client using Hydra capacities (enabled by default in API Platform), you should also `Link`.